modules/src/main/java/org/archive/modules/fetcher/BdbCookieStore.java

```diff
@@ -78,7 +79,7 @@
         @Override public int size() { return wrapped.size(); }
         @Override public boolean isEmpty() { throw new RuntimeException("not implemented"); }
         @Override public boolean contains(Object o) { throw new RuntimeException("not implemented"); }
-        @Override public Iterator<T> iterator() { throw new RuntimeException("not implemented"); }
+        @Override public Iterator<T> iterator() { return (Iterator<T>) Arrays.asList(wrapped.toArray()).iterator(); }
```

Comment:
We should probably also update the class-level javadoc that says iterator() is not implemented, because now it is. :-)

Reply:
Not quite sure how to put it, but I'll have a go!